config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
```

Comment:
Agreed, though now I'm second-guessing myself as to whether a failed check should return `:fail`. It might make more sense to return `:warn` and let the user opt-in to failure with `on_warn: 'fail'`.

As it is, there are several options if someone doesn't care much about docstring conventions. They can:
- Disable the hook entirely
- Set `on_fail: 'warn'`
- Ignore specific errors with `flags: ['--ignore=D101,D202']`, etc